backend/blueprints/spa_api/service_layers/replay/groups.py

```diff
@@ -19,30 +24,59 @@ def __init__(self, title: str, chart_data_points: List[ChartDataPoint], type_: s
     @staticmethod
     def create_from_ids(ids: List[str]) -> List['ReplayGroupChartData']:
         stats = wrapper.get_group_stats(ids)
-
-        players = list(stats['playerStats'].keys())
+        player_stats = stats['playerStats']
+        player_names = [player['name'] for player in player_stats]
         if 'ensembleStats' in stats:
-            players.append('ensembleStats')
+            player_stats.append(stats['ensembleStats'])
+        
+        categories = list(player_stats[0]['stats'].keys())
+
         all_chart_data = []
+        
+        # player_stats_metadata_list = [chart_metadata.stat_name for chart_metadata in player_stats_metadata]
```

Comment:
Is there a reason this code exist but commented?
Can we remove it?

Reply:
I left it in there in case i needed it in the next time around. But it is now gone with https://github.com/SaltieRL/DistributedReplays/pull/283/commits/bea68713019b2eda26eabdb677c8f5946d6ed234